x-pack/plugins/security_solution/public/actions/filter/lens/create_action.ts

```diff
@@ -72,6 +74,7 @@ export const createFilterLensAction = ({
       isInSecurityApp(currentAppId),
     execute: async ({ data }) => {
       const field = data[0]?.columnMeta?.field;
+      const valueType = data[0]?.columnMeta?.sourceParams?.type;
```

Comment:
NIT: Since we are not doing anything else with `valueType`, would it make more sense to store a boolean directly like:
```suggestion
      const isCounter = data[0]?.columnMeta?.sourceParams?.type === 'value_count';
```

So down there the conditional is just:
`if (isCounter) { ...`